src/app/mv-wrapper/media-viewer-wrapper.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Compiler, ViewChild, ViewContainerRef, ComponentFactoryResolver,
+  ComponentFactory, ElementRef, OnInit, Renderer2, Inject, PLATFORM_ID } from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
+import { WindowService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services/window';
+
+const MEDIA_VIEWER = 'media-viewer';
+
+@Component({
+  selector: 'ccd-media-viewer',
+  templateUrl: './media-viewer-wrapper.component.html'
+})
+export class MediaViewerWrapperComponent implements OnInit {
+
+  mediaURL = '';
+  mediaFilename = '';
+  mediaContentType = '';
+
+  public constructor(public renderer: Renderer2,
+                     private el: ElementRef,
+                     private windowService: WindowService,
+                     private readonly viewContainerRef: ViewContainerRef,
+                     private componentFactoryResolver: ComponentFactoryResolver,
+                     @Inject(PLATFORM_ID) private platformId: Object) {
+  }
+
+  async ngOnInit() {
+
+    console.log('Platform ID is ' + this.platformId);
+
+    if (!isPlatformBrowser(this.platformId)) {
+      console.log('On server. Skipping.');
```

Comment:
for my knowledge, does the MV works with both SSR enabled and disabled?

Reply:
MV works only with lazy loading, so this piece of code had turned obsolete.